.github/CODEOWNERS

```diff
@@ -39,14 +39,14 @@
 /StRoot/StDetectorDbMaker/St_tpcSCGLC*             @genevb
 /StRoot/StDetectorDbMaker/St_trigDetSums*          @genevb
 /StRoot/StDetectorDbMaker/St_vertexSeed*           @genevb
-/StRoot/StEEmc*                                    @kkauder           @rkunnawa
-/StRoot/StEEmcSimulatorMaker                       @kkauder           @rkunnawa    @klendathu2k
-/StRoot/StEEmcUtil                                 @kkauder           @rkunnawa    @klendathu2k @tinglin-physics
+/StRoot/StEEmc*                                    @kkauder           @rkunnawa    @Navagyan
+/StRoot/StEEmcSimulatorMaker                       @kkauder           @rkunnawa    @klendathu2k @Navagyan
+/StRoot/StEEmcUtil                                 @kkauder           @rkunnawa    @klendathu2k @tinglin-physics  @Navagyan
 /StRoot/StEEmcPool                                 @tinglin-physics
```

Comment:
```suggestion
/StRoot/StEEmc*                                    @kkauder           @rkunnawa    @Navagyan    @zlchang
/StRoot/StEEmcSimulatorMaker                       @kkauder           @rkunnawa    @klendathu2k @Navagyan         @zlchang
/StRoot/StEEmcUtil                                 @kkauder           @rkunnawa    @klendathu2k @tinglin-physics  @Navagyan      @zlchang
/StRoot/StEEmcPool                                 @tinglin-physics   @zlchang
```

Reply:
Not sure if @Navagyan wants to maintain EEMC. The next EEMC software coordinator is @zlchang.